src/core/qgsrenderedcalculatedresults.h

```diff
@@ -0,0 +1,88 @@
+/***************************************************************************
+    qgsrenderedcalculatedresults.h
+    ----------------
+    copyright            : (C) 2024 by Jean Felder
+    email                : jean dot felder at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSRENDEREDCALCULATEDRESULTS_H
+#define QGSRENDEREDCALCULATEDRESULTS_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgsrendereditemdetails.h"
+
+#include <limits>
+
+
+/**
+ * \ingroup core
+ * \brief Contains computed information about rendered features.
+ * \since QGIS 3.42
+ */
+class CORE_EXPORT QgsRenderedCalculatedResults: public QgsRenderedItemDetails
+{
+  public:
+
+    /**
+     * Constructor for QgsRenderedCalculatedResults.
+     */
+#ifndef SIP_RUN
+    QgsRenderedCalculatedResults( const QString &layerId, double min = std::numeric_limits<double>::quiet_NaN(), double max = std::numeric_limits<double>::quiet_NaN() );
+#else
+    QgsRenderedCalculatedResults( const QString &layerId, SIP_PYOBJECT min SIP_TYPEHINT( Optional[float] ) = Py_None, SIP_PYOBJECT max SIP_TYPEHINT( Optional[float] ) = Py_None ) [( const QString & layerId, double min = 0.0, double max = 0.0 )];
+    % MethodCode
+    double minP = a1 == Py_None ? std::numeric_limits<double>::quiet_NaN() : PyFloat_AsDouble( a1 );
+    double maxP = a2 == Py_None ? std::numeric_limits<double>::quiet_NaN() : PyFloat_AsDouble( a2 );
+    sipCpp = new sipQgsRenderedCalculatedResults( *a0, minP, maxP );
+    % End
+#endif
+
+#ifdef SIP_RUN
+    SIP_PYOBJECT __repr__();
+    % MethodCode
+    QString str = QStringLiteral( "<QgsRenderedCalculatedResults: %1 (%2 x %3)>" ).arg( sipCpp->layerId(), QString::number( sipCpp->min() ), QString::number( sipCpp->max() ) );
+    sipRes = PyUnicode_FromString( str.toUtf8().constData() );
+    % End
+#endif
+
+    /**
+     * Returns the minimum value of the computed information.
+     * \see setMin()
+     */
+    double min() const;
```

Comment:
```suggestion
    double minimum() const;
```

(Also for other members here). Our coding convention is to avoid abbreviations wherever possible, as it makes the API less predictable and less friendly to non-native English speakers.

Reply:
Fixed.